### PR TITLE
Change CI to use build.sh on Linux

### PR DESCRIPTION
**Pull request by iamjasonp**
This change allows us to use build.sh on Linux rather than using build.cmd on
a Windows machine and then using a build flow to propagate the binaries to a
Linux box for testing

It allows us to run CI against published packages rather than built packages
from the latest CI run from corefx

### CODE REVIEW

**Comment by iamjasonp** [comment]
Reopened after #800 seemed to get stuck...

**Comment by iamjasonp** [comment]
@dotnet-bot test ci please

**Comment by iamjasonp** [comment]
@StephenBonikowsky, @mmitche, can you please take a look?

**Comment by mmitche** [comment]
@iamjasonp Will take a look.

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
LGTM

**Comment by iamjasonp** [comment]
@dotnet-bot test this please

File: netci.groovy [hunk 1]
```diff
@@ -229,7 +109,7 @@ build(params + [WCF_LINUX_BUILD_NUMBER: linuxBuildJob.build.number],
 // **************************
 
 // Define build string
-def codeCoverageBuildString = '''build.cmd /p:Coverage=true /p:WithCategories=OuterLoop'''
+def codeCoverageBuildString = '''build.cmd /p:OSGroup=Windows /p:Coverage=true /p:WithCategories=\"\\\"InnerLoop;OuterLoop\\\"\"'''
```
Comment (StephenBonikowsky) [inline netci.groovy]: Have you tried running code coverage runs like this using both Inner and Outer loop tests?

**Comment by StephenBonikowsky** [comment]
LGTM

**Pull request merged**